hack/crossplane/apis/resources.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: Supported Resources
 ---
+These are the CrossPlane committed to our **main** branch, grouped by the originating ARM service.
```

Comment:
minor: This reads weird. These are the Crossplane _resources_?

Also it's `Crossplane` not `CrossPlane` I believe.

Reply:
Amended wording and fixed spelling.